src/components/Integrations/Shopify/Shopify.js

```diff
@@ -61,35 +62,66 @@ const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
       </tbody>
     </table>
   );
+
+  const shopifyHeader = (
+    <>
+      <div className="block">
+        <h2>{_('shopify.header_title')}</h2>
+        <p>
+          <FormattedHTMLMessage id="shopify.header_subtitle" />
+        </p>
+      </div>
+      <hr />
+    </>
+  );
+
+  function useInterval(callback, delay) {
+    const savedCallback = useRef();
+
+    useEffect(() => {
+      savedCallback.current = callback;
+    }, [callback]);
```

Comment:
What is the difference in the behavior of simply using this:

```diff
-    useEffect(() => {
-      savedCallback.current = callback;
-    }, [callback]);
+ savedCallback.current = callback;
```

The last assignment will only have effect when `savedCallback.current != callback`

Reply:
As I've understanded, the callback needs to be new each time, to allow accessing the current states variables, else they get stuck in the first variable.

Reply:
Mmmm, but the `useInterval` function is called each time.

When you have time, let us talk about it, I think that it would be better to understand for me.